stm32-metapac-gen/src/lib.rs

```diff
@@ -443,19 +437,20 @@ pub fn gen(options: Options) {
                     if let Some(clock_prefix) = clock_prefix {
```

Comment:
`clock_prefix` is now completely unused, it should be removed. It still has the effect of skipping the search if peripheral has no clock, but I think always doing the bit search should be fine.